reactDashboard/src/components/DashboardNavbar/DashboardNavbar.css

```diff
@@ -1,64 +1,126 @@
-.dashboard-navbar {
-  margin: 0 1px 4px 1px;
+.darkmode-btn-toggle {
+  background-color: #287db2;
+  border-radius: 50%;
+
+  padding: 6px;
+  font-size: 25px;
+  margin: auto 0 auto auto;
 }
 
-.dashboard-navbar img#logo {
-  height: 100px;
-  margin-right: 20px;
+.darkmode .darkmode-btn-toggle {
+  color: #fd8d3c;
+  background-color: #121212;
 }
 
-.dashboard-navbar .heading-container {
-  display: flex;
-  flex-direction: row;
-  padding: 5px 0 5px;
-  align-items: center;
-  justify-content: space-between;
-  flex-grow: 1;
+.darkmode-btn-toggle:hover {
+  background-color: #c1def1;
+  cursor: pointer;
 }
 
-.dashboard-navbar .heading-container.collapse:not(.show) {
-    display: none;
+.darkmode .darkmode-btn-toggle:hover {
+  color: #121212;
+  background-color: #fd8d3c;
 }
 
-.dashboard-navbar .heading {
-  display: flex;
-  align-items: center;
-  padding-right: 20px;
+.hide-darkmode .darkmode-btn-toggle {
+  display: none;
 }
 
-.dashboard-navbar .heading .h1 {
-  display: inline-block;
+.hide-darkmode {
+  display: flex;
+  flex-direction: row;
 }
 
 .navbar-links {
-  display: flex;
-  justify-content: flex-end;
-  padding: 1.25rem 0rem !important;
+  margin-right: 0;
+  margin-bottom: 0;
 }
 
 .nav-link {
-  font-size: 1.5rem;
-  font-weight: 600;
-  margin: 10px 0;
-  padding: 0 1.75rem !important;
-  flex-basis: 0;
-  border-left: 2px solid #3c3c3b;
+  font-size: 1em;
+  width: 180px;
+  padding-top: 10px;
+  margin-top: 10px;
+  padding-bottom: 20px;
+  text-align: center;
+  font-weight: bold;
 }
 
-.nav-link:hover {
+.navbar-links a.nav-link:hover {
   text-decoration: underline;
+  color: #121212;
+}
+
+.navbar-links a.nav-link.selected {
+  color: #121212;
+  background-color: #f2f2f2;
+  text-decoration: none;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.darkmode .navbar-links a.nav-link.selected {
+  background-color: white;
+}
+
+.dashboard-navbar img#logo {
+  height: 60px;
+  margin: 10px 10px 10px 0;
+  padding: 0;
+}
+
+.darkmode .dashboard-navbar img#logo {
+  background-color: #f2f2f2;
+}
+
+.dashboard-navbar {
+  margin: 0 15px;
+  padding: 30px 10px 0;
+  background-color: white;
+}
+
+.darkmode .dashboard-navbar {
+  background-color: #f2f2f2;
+  opacity: 0.75;
```

Comment:
Adjust the background-color rather than merge with whatever is underneath.

Reply:
✅  😎